### PR TITLE
feat-be: Implement QR Code interaction

### LINKED ISSUE #100: feat-be: Add simple interaction with QR code

### PR DESCRIPTION
**Pull request by patrykferenc**
Simple interaction, please check out the based image in resources.

Added diagrams to VP our beloved.

Closes #100

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -48,3 +48,10 @@ services:
     extends:
       file: ./payments-be/compose.yaml
       service: payments-db
+
+  # game
+  game-service:
+    extends:
+      file: ./game-service/compose.yaml
```
Comment (iruszpel) [inline compose.yaml:55]: Should be `./game-service-be/compose.yaml`

File: game-service-be/compose.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+services:
+  game-service:
+    image: larpex/game-service:latest
+    ports:
+      - "8085:8080"
```
Comment (iruszpel) [inline game-service-be/compose.yaml:5]: ` Ports are not available: exposing port TCP 0.0.0.0:8085 -> 0.0.0.0:0: listen tcp 0.0.0.0:8085: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.`

8085 seems to conflict with events service

**Comment by iruszpel** [comment]
Also it's worth updating `openapi.yaml` in gateway

**Review by bastyje — CHANGES_REQUESTED** [review]
fix issues pointed by @iruszpel and we're good to go

File: game-service-be/compose.yaml (hunk 2 above)
Comment (patrykferenc) [reply game-service-be/compose.yaml:5]: jeez I didn't know events service takes up 3 ports xdddd that's insane

Comment (patrykferenc) [reply game-service-be/compose.yaml]: fixed

**Comment by patrykferenc** [comment]
> Also it's worth updating `openapi.yaml` in gateway
> fix issues pointed by @iruszpel and we're good to go
Done.

File: compose.yaml (hunk 1 above)
Comment (patrykferenc) [reply compose.yaml]: done

**Review by bastyje — APPROVED** [review]

**Pull request merged**